:100644 100644 8d2b552... 0000000... M	CMakeLists.txt
:100644 100644 beb259a... 0000000... M	googlemock/CMakeLists.txt
:100644 100644 621d0f0... 0000000... M	googletest/CMakeLists.txt

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8d2b552..5c0d122 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,3 @@
-cmake_minimum_required(VERSION 2.6.2)
-
 project( googletest-distribution )
 
 enable_testing()
diff --git a/googlemock/CMakeLists.txt b/googlemock/CMakeLists.txt
index beb259a..6f0cb3b 100644
--- a/googlemock/CMakeLists.txt
+++ b/googlemock/CMakeLists.txt
@@ -7,7 +7,7 @@
 
 # BUILD_SHARED_LIBS is a standard CMake variable, but we declare it here to
 # make it prominent in the GUI.
-option(BUILD_SHARED_LIBS "Build shared libraries (DLLs)." OFF)
+set(BUILD_SHARED_LIBS OFF)
 
 option(gmock_build_tests "Build all of Google Mock's own tests." OFF)
 
@@ -38,7 +38,6 @@ endif()
 # ${gmock_BINARY_DIR}.
 # Language "C" is required for find_package(Threads).
 project(gmock CXX C)
-cmake_minimum_required(VERSION 2.6.2)
 
 if (COMMAND set_up_hermetic_build)
   set_up_hermetic_build()
@@ -103,10 +102,10 @@ endif()
 ########################################################################
 #
 # Install rules
-install(TARGETS gmock gmock_main
-  DESTINATION lib)
-install(DIRECTORY ${gmock_SOURCE_DIR}/include/gmock
-  DESTINATION include)
+#install(TARGETS gmock gmock_main
+#  DESTINATION lib)
+#install(DIRECTORY ${gmock_SOURCE_DIR}/include/gmock
+#  DESTINATION include)
 
 ########################################################################
 #
diff --git a/googletest/CMakeLists.txt b/googletest/CMakeLists.txt
index 621d0f0..c77f9ef 100644
--- a/googletest/CMakeLists.txt
+++ b/googletest/CMakeLists.txt
@@ -7,7 +7,7 @@
 
 # BUILD_SHARED_LIBS is a standard CMake variable, but we declare it here to
 # make it prominent in the GUI.
-option(BUILD_SHARED_LIBS "Build shared libraries (DLLs)." OFF)
+set(BUILD_SHARED_LIBS OFF)
 
 # When other libraries are using a shared version of runtime libraries,
 # Google Test also has to use one.
@@ -45,7 +45,6 @@ endif()
 # ${gtest_BINARY_DIR}.
 # Language "C" is required for find_package(Threads).
 project(gtest CXX C)
-cmake_minimum_required(VERSION 2.6.2)
 
 if (COMMAND set_up_hermetic_build)
   set_up_hermetic_build()
@@ -102,10 +101,10 @@ endif()
 ########################################################################
 #
 # Install rules
-install(TARGETS gtest gtest_main
-  DESTINATION lib)
-install(DIRECTORY ${gtest_SOURCE_DIR}/include/gtest
-  DESTINATION include)
+#install(TARGETS gtest gtest_main
+#  DESTINATION lib)
+#install(DIRECTORY ${gtest_SOURCE_DIR}/include/gtest
+#  DESTINATION include)
 
 ########################################################################
 #
