diff --git a/googlemock/include/gmock/gmock-matchers.h b/googlemock/include/gmock/gmock-matchers.h
index 33b37a7..fbb5734 100644
--- a/googlemock/include/gmock/gmock-matchers.h
+++ b/googlemock/include/gmock/gmock-matchers.h
@@ -1620,8 +1620,13 @@ class VariadicMatcher {
   }
 
  private:
+#if defined(__GNUC__) && !defined(__clang__)
+#pragma GCC diagnostic ignored "-Wconversion"
+#endif
   typedef MatcherList<sizeof...(Args), Args...> MatcherListType;
-
+#if defined(__GNUC__) && !defined(__clang__)
+#pragma GCC diagnostic warning "-Wconversion"
+#endif
   const typename MatcherListType::ListType matchers_;
 
   GTEST_DISALLOW_ASSIGN_(VariadicMatcher);
